### DISCUSSION TITLE
Model quantization

### DISCUSSION THREAD

**Discussion by AndreiNanescu** [microsoft/onnxruntime#17906] [Other Q&A]
Hello, I have a question regarding model quantization. I am currently using an ORT format model and would like to know if the quantization tools work with it, or if I need to quantize the ONNX model and then convert it to the ORT format. Thanks!

**Comment by faxu** [comment]
Are you using ORT format model for a mobile scenario? 
CC @skottmckay

**Comment by skottmckay** [comment]
Converting to an ORT format model is the last step for mobile scenarios where binary size is critical. The tools work on ONNX models to be more general purpose, so please quantize the ONNX model and convert to ORT format after that as needed.
